doc/source/install.txt

```diff
@@ -1,53 +1,153 @@
-Pre-built installation
+Installing scikit-image
+-----------------------
+If you are on Mac OS X you're lucky, open the terminal and install
```

Comment:
But joking aside, in this particular case, if you are on OSX, you are lucky.   Not a value judgement about operating systems, just a comment on the convenience of pip installs.